system-addon/lib/ActivityStreamMessageChannel.jsm

```diff
@@ -136,8 +136,8 @@ this.ActivityStreamMessageChannel = class ActivityStreamMessageChannel {
     this.channel.addMessageListener(this.incomingMessageName, this.onMessage);
 
     // Some pages might have already loaded, so we won't get the usual message
-    for (const {loaded, portID} of this.channel.messagePorts) {
-      const simulatedMsg = {target: {portID}};
+    for (const {browser, loaded, portID} of this.channel.messagePorts) {
+      const simulatedMsg = {target: {browser, portID}};
       this.onNewTabInit(simulatedMsg);
       if (loaded) {
```

Comment:
When https://bugzilla.mozilla.org/show_bug.cgi?id=1398955 lands, you can access `.url` from the message port. Probably clean this up to just do
```js
for (const target of this.channel.messagePorts) {
  const simulatedMsg = {target};
  …
  if (target.loaded) {
    …
```